Podcast/EpisodeTableViewCell.swift

```diff
@@ -304,6 +282,49 @@ class EpisodeTableViewCell: UITableViewCell {
         cardID = episode.id
     }
     
+    func setupTagButtons(episode: Episode) {
```

Comment:
because they are two different cells ... with different views, etc. Where would you suggest I put it?